plugins/sqs/doc/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+find_package(Doxygen)
+if(NOT DOXYGEN_FOUND)
+    message("Doxygen not found, I will not generate/install the documentation")
+else()
+   configure_file(Doxyfile.in Doxyfile)
```

Comment:
I think this means Doxyfile.in is generated by the build process - if so it should not have been checked in. Best to create it outside the source tree too...